runner/src/server/plugins/engine/pageControllers/PageControllerBase.ts

```diff
@@ -555,11 +558,29 @@ export class PageControllerBase {
 
       viewModel.backLink =
         state.callback?.returnUrl ?? progress[progress.length - 2];
-      viewModel.backLinkText =
-        this.model.def?.backLinkText ?? "Go back to application overview";
+      if (state["metadata"]["has_eligibility"]) {
+        viewModel.backLinkText = this.getBackLinkText(true);
+      } else {
+        viewModel.backLinkText =
+          this.model.def?.backLinkText ?? this.getBackLinkText(false);
+      }
       return h.view(this.viewName, viewModel);
     };
   }
+  getBackLinkText(eligibility: boolean) {
+    if (eligibility) {
```

Comment:
Same like above 

Can we have a util to do these conversions then once we have a initiative to move it to a i8n we just wanted to update that place ?